rxandroidble/src/main/java/com/polidea/rxandroidble/ClientComponent.java

```diff
@@ -131,6 +133,19 @@ static int provideDeviceSdk() {
             return Build.VERSION.SDK_INT;
         }
 
+        @Provides
+        ContentResolver provideContentResolver() {
+            return context.getContentResolver();
+        }
+
+        @Provides
+        @Named(PlatformConstants.STRING_LOCATION_SERVICES_CHANGED_INTENT_ACTION)
+        static String provideLocationServicesChangedIntentAction() {
```

Comment:
I'm always hesitant when I see this. Is it a dependency or part of the logic? If a dependency then it's a good idea to inject, if a logic then not. 

I think that SDK_INT is a dependency here.

Reply:
Do you know any good patterns to distinguish?